src/network/clientpackethandler.cpp

```diff
@@ -821,6 +826,26 @@ void Client::handleCommand_StopSound(NetworkPacket* pkt)
 	}
 }
 
+void Client::handleCommand_FadeSound(NetworkPacket* pkt)
```

Comment:
Pointer on the wrong side, they belong to the variable name. `NetworkPacket *pkt`